v2/trace/ddpgx/trace.go

```diff
@@ -0,0 +1,46 @@
+package ddpgx
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	dd_ext "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	dd_tracer "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func argsToAttributes(args ...interface{}) map[string]interface{} {
+	output := map[string]interface{}{}
+
+	for i := range args {
+		key := fmt.Sprintf("sql.args.%d", i)
+		output[key] = args[i]
+	}
+
+	return output
+}
+
+func tryTrace(ctx context.Context, startTime time.Time, driver, serviceName, resource string, metadata map[string]interface{}, err error) {
```

Comment:
wrong order: `driver, serviceName, resource` in regards to how the func is used
maybe consider using a struct instead